src/components/ui/Suggest.jsx

```diff
@@ -30,6 +30,14 @@ const Suggest = ({
     setItems([]);
   };
 
+  useEffect(() => {
+    if (isOpen) {
+      document.body.classList.add('suggestions-open');
```

Comment:
Why is it needed ?

Reply:
To hide the panel, here https://github.com/QwantResearch/erdapfel/pull/691/files#diff-46b992b598e848147f038499405bff57R1

Reply:
Ok, could we come up with a solution which would not involve adding a class in the suggest component ? Maybe using an event (Suggest.js should not know any panel's implementation), a "global store" or simply using a "panel" lib which would expose a "toggle" function ?

Reply:
> I think Suggest.jsx should not know nothing about panel's implementation.

Technically it knows nothing here, it just sets a state on the document in a clearly separated side-effect, wuthout having to now what's made of it. Actually using an imperative panel toggling lib would be more coupled than that :)

> Maybe using an event, a "global store" or simply using a "panel" lib which would expose a "toggle" function ?

I chose this class solution because this is the easier one to toggle the visibility in CSS only. I don't want to toggle panels with conditional rendering using React, as it won't be as fast as needed and some panels will request some info when remounting.

Sure I could use an event, but the listener code would probably be the same. Would you like it better?

Reply:
The only thing bothering me is that I would prefer a (mini) layer of abstraction, I would prefer not having to know a "suggestions-open" class has to be set/unset inside `Suggest.jsx` directly, because if we change the way we hide the panel one day, we would need to edit `Suggest.jsx` instead of a file like `lib/panel.js` for instance.

Reply:
I understand. So, what do you think about firing an event like `'toggle-suggestions'` in `Suggest.jsx` and listening to it in `PanelManager.jsx`? I can't see a better place to "manage panels" :smiley:

Reply:
I think we can avoid events, what about creating a `lib/panel.js` with two exported functions: `open` `close` (or hide).
Each one of these function would set/unset the class `suggestions-open`. This way, any part or our app wanting to open/hide the panel would call `panel.close()` or `panel.open()`.